pkg/template/helper.go

```diff
@@ -42,3 +44,35 @@ func (r TemplateReference) String() string {
 	}
 	return r.Name
 }
+
+// ConvertUserToTemplateInstanceRequester copies analogous fields from user.Info to TemplateInstanceRequester
+func ConvertUserToTemplateInstanceRequester(u user.Info) templateapi.TemplateInstanceRequester {
```

Comment:
@bparees and I discussed at my cube ... I believe we are in sync wrt a) not embedding info.User in the TemplateInstanceRequester struct (protobuf gen) and b) why the osg orig id hdr json cannot be marshaled into a info.User, and needs to marshaled into a TemplateInstanceRequester

hence the presence of these Covert methods